src/components/personalBankAccountPropTypes.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    /** General errors */
+    error: PropTypes.string,
+
+    /** Field specific errors */
+    errorFields: PropTypes.shape({
+        /** Plaid selector and password fields */
+        plaidSelector: PropTypes.shape({
+            /** Key: timestamp - Value: Error message */
+        }),
+    }),
+
+    pendingFields: PropTypes.shape({
```

Comment:
Are you sure this is how it's supposed to be?
Looking at existing code, pendingFields seems to be always a map of fieldName to redBrickRoad action?

Reply:
Updated